src/dictionaries.cpp

```diff
@@ -21,6 +21,12 @@ void BasicDictionary::add(uint32_t code, const std::string& name)
     m_reverseDict.emplace(name, code);
 }
 
+void BasicDictionary::append(const BasicDictionary& basicDict)
+{
+    m_rightDict.insert(basicDict.m_rightDict.begin(), basicDict.m_rightDict.end());
+    m_reverseDict.insert(basicDict.m_reverseDict.begin(), basicDict.m_reverseDict.end());
```

Comment:
The idea is correct, but there is a problem with this solution. If we are inserting a value that already exists in our dictionary, it won't be inserted or assigned. E.g., if we have a mapping
```
Some-Attribute 5
```
and then in the included dictionary we have
```
Some-Attribute 25
```
in the result of the `append` call we would end up with
```
Some-Attribute 5
```
So it does not change.

But we need to be able to override mappings, that the latter value has a precedence over the former.